common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2061,6 +2066,21 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(0)
 
+  val NETWORK_IO_CLIENT_CONFLICT_AVOID_CHOOSER_ENABLE: ConfigEntry[Boolean] =
+    buildConf("celeborn.<module>.io.conflictAvoidChooser.enable")
+      .categories("network")
+      .doc("Whether to use conflict avoid event executor chooser in the client thread pool. " +
```

Comment:
the words need to be extended, it's unclear for users what the "conflict" is, and how it can be resolved by enabling this conf.